Sources/PackageModel/EnabledTraitsMap.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// A wrapper for a dictionary that stores the transitively enabled traits for each package.
+public struct EnabledTraitsMap: ExpressibleByDictionaryLiteral {
```

Comment:
Does this need to be public?

Reply:
I'll double check, it may not need to be